src/hotspot/cpu/x86/sharedRuntime_x86.cpp

```diff
@@ -83,3 +84,35 @@ void SharedRuntime::inline_check_hashcode_from_object_header(MacroAssembler* mas
 }
 #endif //COMPILER1
 
+JRT_LEAF(jfloat, SharedRuntime::frem(jfloat  x, jfloat  y))
```

Comment:
Nit: extra space before x and y.

Reply:
I agree but FYI it was just copy-pasted from existing source:
> `-JRT_LEAF(jfloat, SharedRuntime::frem(jfloat  x, jfloat  y))`